runtime/worker.rs

```diff
@@ -58,6 +61,10 @@ pub struct MainWorker {
   pub js_runtime: JsRuntime,
   should_break_on_first_statement: bool,
   exit_code: ExitCode,
+  js_run_tests_callback: v8::Global<v8::Function>,
+  js_run_benchmarks_callback: v8::Global<v8::Function>,
+  js_enable_test_callback: v8::Global<v8::Function>,
+  js_enable_bench_callback: v8::Global<v8::Function>,
```

Comment:
Could all of this be moved to `cli/worker.rs`?

Reply:
It can, but the intention is to add lower level `deno_runtime` bindings below these. For now this is just rusting the API exposed from `runtime/js/40_testing.js`. Later I want to take that further and make the "runtime" side of the test runner more self-contained and embeddable. This is laid out between the TODOs and PR description